src/main/resources/static/client.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+    <script src="https://cdn.jsdelivr.net/npm/@stomp/stompjs@7.0.0/bundles/stomp.umd.min.js"></script>
+    <script src="https://d3js.org/d3.v7.min.js"></script>
+    <script src="/clientStompManager.js"></script>
+    <script src="/d3manipulation.js"></script>
+</head>
+<body>
+<svg id="svg"></svg>
+<div id="buttonsDiv">
+    <button class="answerButton">answerButton1</button>
+    <button class="answerButton">answerButton2</button>
+    <button class="answerButton">answerButton3</button>
+</div>
+<label style="width: 100px;" id="questionNameLabel">No question</label>
+<!--https://stackoverflow.com/a/15069289-->
+<script>
+    connectStompClient();
+
+    document.addEventListener('DOMContentLoaded', function() {
+        var svgElement = document.getElementById("svg");
+        const params = new URLSearchParams(window.location.search);
+        var initialQuestion;
+        if (params.size === 0) {
+            // voting client
+            setUpSvg(svgElement, window.innerWidth - 25, window.innerHeight - 75);
```

Comment:
why 25 and 75 ? maybe a constant can explain ?

Reply:
Defined constants to identify padding values.